testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Hadoop.java

```diff
@@ -66,8 +68,8 @@ public void extendEnvironment(Environment.Builder builder)
     {
         builder.addContainer(createHadoopContainer(dockerFiles, portBinder, hadoopBaseImage + ":" + hadoopImagesVersion, HADOOP));
 
-        builder.configureContainer(
-                COORDINATOR,
+        builder.configureContainersIfPresent(
+                ImmutableSet.of(COORDINATOR, WORKER),
```

Comment:
I like the syntax, but there is one downside:

- we know COORDINATOR _should_ exists, so it's actually an error if it doesn't
- we want to apply this is conditionally to WORKER, if it exists

maybe 

```
configureContainers(Set<String> containersToConfigure, Set<String> optionalContainersToConfigure, Consumer)
```